aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -888,6 +896,119 @@ module aptos_framework::delegation_pool {
         }
     }
 
+    /// Initialize a delegation pool from an existing owned staking contract.
+    /// The newly created delegation pool will preserve the operator and commission fee. The staker becomes the pool
+    /// owner and has the same delegated voter set.
+    public entry fun initialize_delegation_pool_from_staking_contract(
+        staker: &signer,
+        operator: address,
+    ) acquires DelegationPool, GovernanceRecords, BeneficiaryForOperator, NextCommissionPercentage {
+        assert!(features::delegation_pools_enabled(), error::invalid_state(EDELEGATION_POOLS_DISABLED));
```

Comment:
We likely don't need this anymore since delegation pool has been live for almost 2 years now and will never be disabled